client/src/App.jsx

```diff
@@ -61,6 +63,13 @@ const App = () => {
                 <Route path="/shares">
                   <Shares />
                 </Route>
+                <Route path="/profile">
+                  <Profile auth={auth} />
```

Comment:
No passing props is needed for auth. Use useContext hook to retrieve it
```suggestion
                  <Profile />
```